### PR TITLE
Chips

### PR DESCRIPTION
**Pull request by andrewholsted**
This adds styles and docs for the chip component. 

My css foo is circa 2012. I didn't use variables for colors, I assume that's a no-no? What other improvements can I make?

### CODE REVIEW

File: scss/components/chip.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/**
+* @name Chip
+* @category containers
+* @component chip
+*/
+
+.cui-chip {
+  align-items: center;
+  display: flex;
```
[the remark this answers is not in the document]
Comment (andrewholsted) [reply scss/components/chip.scss]: ooo. nice. I'll take a look at that

File: scss/components/chip.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
+/**
+* @name Chip
+* @category containers
+* @component chip
+*/
+
+.cui-chip {
```
[the remark this answers is not in the document]
Comment (pauljeter) [reply scss/components/chip.scss]: At the very least, we should prefix with the cui variable. 
```scss
.#{$cui}-chip
```

[the remark this answers is not in the document]
Comment (pauljeter) [reply scss/components/chip.scss]: fyi, the exports mixin ensures that the css is only included once in the final output, even if it is included more than once.

File: scss/components/chip.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  align-items: center;
+  display: flex;
+  max-width: 400px;
+  background-color: #F4F3F4;
+  border-radius: 4px;
+
+  // When aligning vertically with flexbox, it appears
+  // all the svgs aren't centered in the middle. This is not
+  // a good fix.
+  svg {
```
[the remark this answers is not in the document]
Comment (pauljeter) [reply scss/components/chip.scss]: 5px doesn't sound like the offset issue. Does the same happen if you use the font?
`<i class="icon icon-whatever" />

File: scss/components/chip.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* @name Chip
+* @category containers
+* @component chip
+*/
+
+.cui-chip {
+  align-items: center;
+  display: flex;
+  max-width: 400px;
+  background-color: #F4F3F4;
```
Comment (pauljeter) [inline scss/components/chip.scss]: You already mentioned it, but please use variables for colors.

File: scss/components/chip.scss [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  border-radius: 4px;
+
+  // When aligning vertically with flexbox, it appears
+  // all the svgs aren't centered in the middle. This is not
+  // a good fix.
+  svg {
+    margin-top: 5px
+  }
+
+  .cui-chip-left{
```
Comment (pauljeter) [inline scss/components/chip.scss]: You can avoid repeating the ".cui-chip" on all of these nested classes by using `&-left`, `&-center`, etc.

File: scss/components/chip.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
+/**
+* @name Chip
+* @category containers
+* @component chip
+*/
+
+@import '../settings/core';
+@import '../settings/colors';
+
+.#{$cui}-chip {
```
Comment (bfbiggs) [inline scss/components/chip.scss]: I agree with @pauljeter that this is a great first PR but that being said it's missing some of the basic requirements:
``` 
@include exports('cui-chip') {
}
```

File: scss/components/chip.scss [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  max-width: 400px;
+  background-color: $gray-6;
+  border-radius: 4px;
+  @include flex;
+
+  &-left {
+    width: 20%;
+    text-align: center;
+    padding: 10px;
+    @include flex;
```
Comment (bfbiggs) [inline scss/components/chip.scss]: this file needs to import the flex mixin file

File: scss/components/chip.scss (hunk 6 above)
Comment (andrewholsted) [reply scss/components/chip.scss]: ah, I had that at one point and I think I removed it inadvertently in a rebase

**Pull request merged**